MobileWallet/Screens/Home/TransactionHistory/TransactionsTableViewController.swift

```diff
@@ -47,6 +47,30 @@ protocol TransactionsTableViewDelegate: class {
 }
 
 class TransactionsTableViewController: UITableViewController {
+
+    enum TransactionsTableViewBackground: Equatable {
```

Comment:
`TransactionsTableViewBackground` looks like a UI component name. Since it's in the class already you can just call it `BackgroundState`.